sale_order_price_recalculation/README.rst

```diff
@@ -0,0 +1,63 @@
+===================================================
+Recalculation of sales order lines prices on demand
+===================================================
+
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
```

Comment:
Indeed it looks weird ...

We could try with a 

    |Licence|

    .. |Licence| image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
       :alt: License: AGPL-3
       :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html